.github/workflows/ci.yml

```diff
@@ -25,6 +25,13 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
           cache: "npm"
-      - run: npm ci
-      - run: npm test
-      - run: npm run lint
+      - name: Install Dependencies
+        run: npm install
+      - name: Lint
+        run: npm run lint
+      - name: Continuous Integration
+        run: npm ci
```

Comment:
npm ci basically installs all dependencies. So you can just run `npm ci` in step one instead of `npm install`.
https://docs.npmjs.com/cli/v8/commands/npm-ci

Reply:
Wasn't sure where to place that one. Thanks!